### PR TITLE
Feature/product

### PR DESCRIPTION
**Pull request by adrianovi**
Created the basic functionality for products (save and get products)

### CODE REVIEW

File: project/store/src/main/java/org/fjala/prog102/store/repositories/ProductRepository.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package org.fjala.prog102.store.repositories;
+
+import org.fjala.prog102.store.models.Product;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface ProductRepository extends JpaRepository<Product, Long> {
```
Comment (adhes14) [inline project/store/src/main/java/org/fjala/prog102/store/repositories/ProductRepository.java:6]: I think we should extend from CrudRepository

**Review by jimytz — APPROVED** [review]
Good job Adrian!

File: project/store/src/main/java/org/fjala/prog102/store/repositories/ProductRepository.java (hunk 1 above)
Comment (jmt4b04d4v) [reply project/store/src/main/java/org/fjala/prog102/store/repositories/ProductRepository.java:6]: JpaRepository extends PagingAndSortingRepository, which in course extends CrudRepository. Please read about both on them and choose the one that fits your needs.
https://www.geeksforgeeks.org/spring-boot-difference-between-crudrepository-and-jparepository/

Comment (adrianovi) [reply project/store/src/main/java/org/fjala/prog102/store/repositories/ProductRepository.java:6]: So I think we just need Crud repository, I'll change it on the next PR

**Pull request merged**